cmd/kpromo/cmd/pr/pr.go

```diff
@@ -140,10 +143,20 @@ func init() {
 		&promoteOpts.images,
 		"image",
 		"",
+		// TODO: Parameterize
 		[]string{""}, // default to a single empty string image filter which will promote all images matching the tag.
 		"image to promote. If not specified, all images matching the tag will be promoted",
 	)
 
+	PRCmd.PersistentFlags().StringSliceVarP(
+		&promoteOpts.digests,
+		"digests",
+		"",
+		// TODO: Parameterize
+		[]string{""}, // default to a single empty string image filter which will promote all images matching the tag.
+		"digests to promote. If not specified, all images matching the tag will be promoted",
```

Comment:
"If not specified, all images matching the tag will be promoted" -> not sure how this translates to digests but it should probably be more specific to digests.

Reply:
Dropped this altogether for both flags.
As we iterate, I'll extend the CLI guidance on the flags, which is more visible than the code comments.